var/spack/repos/builtin/packages/icu4c/package.py

```diff
@@ -12,14 +12,22 @@ class Icu4c(AutotoolsPackage):
     C/C++ interface."""
```

Comment:
Should this package be renamed to `icu`? It looks like the tarball contains both `icu4c` and `icu4j`. Are we building both?

Reply:
I do not think so. There is a separate tarball for icu4j, as well as other resources specific to icu4j.